OpenRA.Game/Actor.cs

```diff
@@ -295,6 +295,11 @@ public bool IsDisabled()
 			return false;
 		}
 
+		public bool IsInvisibleTo(Player player)
```

Comment:
I need the negative selector for filtering in both the ViewportControllerWidget and CanViewActor.